diff --git a/app/app.go b/app/app.go
index 8f143271..551abb63 100644
--- a/app/app.go
+++ b/app/app.go
@@ -961,6 +961,10 @@ func (app *Evmos) BeginBlocker(ctx sdk.Context) (sdk.BeginBlock, error) {
 
 // EndBlocker updates every end block
 func (app *Evmos) EndBlocker(ctx sdk.Context) (sdk.EndBlock, error) {
+	if ctx.BlockHeight()%10 == 0 {
+		store := ctx.KVStore(app.keys["evm"])
+		store.Set([]byte("hello"), []byte("world"))
+	}
 	return app.mm.EndBlock(ctx)
 }
 
