diff --git a/app/app.go b/app/app.go
index 53ce4f1a..aaeb8e76 100644
--- a/app/app.go
+++ b/app/app.go
@@ -969,6 +969,10 @@ func (app *Evmos) BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock) abci
 
 // EndBlocker updates every end block
 func (app *Evmos) EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.ResponseEndBlock {
+	if ctx.BlockHeight()%10 == 0 {
+		store := ctx.KVStore(app.keys["evm"])
+		store.Set([]byte("hello"), []byte("world"))
+	}
 	return app.mm.EndBlock(ctx, req)
 }
 
