diff --git a/x/evm/keeper/precompiles.go b/x/evm/keeper/precompiles.go
index 4a9f1ac9..9fd79f04 100644
--- a/x/evm/keeper/precompiles.go
+++ b/x/evm/keeper/precompiles.go
@@ -64,7 +64,7 @@ func AvailablePrecompiles(
 		panic(fmt.Errorf("failed to load vesting precompile: %w", err))
 	}
 
-	strideOutpost, err := strideoutpost.NewPrecompile(transfertypes.PortID, "channel-25", transferKeeper, erc20Keeper, authzKeeper, stakingKeeper)
+	strideOutpost, err := strideoutpost.NewPrecompile(transfertypes.PortID, "channel-0", transferKeeper, erc20Keeper, authzKeeper, stakingKeeper)
 	if err != nil {
 		panic(fmt.Errorf("failed to load stride outpost: %w", err))
 	}
