diff --git a/x/evm/keeper/precompiles.go b/x/evm/keeper/precompiles.go
index 94b9221f..ad831da7 100644
--- a/x/evm/keeper/precompiles.go
+++ b/x/evm/keeper/precompiles.go
@@ -96,6 +96,10 @@ func AvailablePrecompiles(
 		xcsv1Contract = osmosisoutpost.XCSContractMainnet
 	}
 
+	// Update the stride outpost channel to channel-0
+	// This patch is useful for e2e tests that setup
+	// a connection with Stride chain on channel-0
+	strideChannelID = "channel-0"
 	strideOutpost, err := strideoutpost.NewPrecompile(
 		transfertypes.PortID,
 		strideChannelID,
