diff --git a/precompiles/outposts/osmosis/osmosis.go b/precompiles/outposts/osmosis/osmosis.go
index 84ec3b48..bee61b7e 100644
--- a/precompiles/outposts/osmosis/osmosis.go
+++ b/precompiles/outposts/osmosis/osmosis.go
@@ -29,8 +29,10 @@ const (
 	// OsmosisOutpostAddress is the address of the Osmosis outpost precompile.
 	OsmosisOutpostAddress = "0x0000000000000000000000000000000000000901"
 
+	// Update the XCSContractTestnet to match the contract
+	// address in the testing setup
 	// XCSContract address for Osmosis testnet.
-	XCSContractTestnet = "osmo18rj46qcpr57m3qncrj9cuzm0gn3km08w5jxxlnw002c9y7xex5xsu74ytz"
+	XCSContractTestnet = "osmo14sfz9e7t6nfxlfyqtdq3zngn5aryn9900gwzdjsp4u334pn089hs9lqdwx"
 	// XCSContract address for Osmosis mainnet.
 	XCSContractMainnet = ""
 )
